arroyo/processing/processor.py

```diff
@@ -35,6 +35,7 @@
 
 METRICS_FREQUENCY_SEC = 1.0  # In seconds
 BACKPRESSURE_THRESHOLD = 5.0  # In seconds
+LOGGING_FREQUENCY_SEC = 60.0  # In seconds
```

Comment:
Let's make it longer now that we know what the likely issue is. 
64 consumers would log this every second. 
What about once every 2-3 minutes ?